docker/docker_image_src.go

```diff
@@ -73,10 +71,10 @@ func newImageSource(ctx context.Context, sys *types.SystemContext, ref dockerRef
 
 // newImageSourceAttempt is an internal helper for newImageSource. Everyone else must call newImageSource.
 // Given a specified source and primaryDomain, return a dockerImage source, or an error, and an indication whether the error is fatal.
```

Comment:
Non-blocking nit: the comment needs an update.

Reply:
Fixed, thanks!